Neos.Eel/Classes/Helper/ArrayHelper.php

```diff
@@ -30,16 +31,18 @@ class ArrayHelper implements ProtectedContextAwareInterface
     /**
      * Concatenate arrays or values to a new array
      *
-     * @param array|mixed $array1 First array or value
-     * @param array|mixed $array2 Second array or value
-     * @param array|mixed $array_ Optional variable list of additional arrays / values
+     * @param iterable|mixed $array1 First array or value
+     * @param iterable|mixed $array2 Second array or value
+     * @param iterable|mixed $array_ Optional variable list of additional arrays / values
      * @return array The array with concatenated arrays or values
      */
     public function concat($array1, $array2, $array_ = null): array
```

Comment:
Note to self, we could make this into a variadic function instead of using func_get_args, but unrelated to this change...